packages/next/build/index.ts

```diff
@@ -739,6 +739,13 @@ export default async function build(
         )
       }
 
+      const breakingChangeIndex = error.indexOf(
+        '\n\nBREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.'
```

Comment:
@sokra is there a better way to exclude the compilation from showing these, an option maybe?

Reply:
looks there is no other way as far as I checked https://github.com/webpack/webpack/blob/main/lib/ModuleNotFoundError.js#L63